Make maven invocations quiet

tautschnig opened the pull request:
The status messages make it unnecessarily hard to spot other compilation errors.
Also disable the attempt to run tests as there aren't any.

<!---
Thank you for your contribution. Please make sure your pull request fulfils all of the below requirements. If you cannot currently tick all the boxes, but would still like to create a PR, then add the label "work in progress" and assign the PR to yourself.
--->

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a My contribution is formatted in line with CODING_STANDARD.md.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

<!---
See, e.g., https://chris.beams.io/posts/git-commit/ for general guidelines on commit messages.

If you have created commits mixing multiple features and/or unrelated white-space changes, use a sequence involving git reset and git add -p to fix this.
--->

chrisr-diffblue reviewed (approved):
Tentatively approving this, but I'm not entirely sure what testing, if any, will be disabled by `-Dmaven.test.skip=true`

tautschnig commented:
@chrisr-diffblue Once we actually have tests, we should revisit disabling the tests. Right now this really just silences the useless output that 0 out of 0 tests have been run.

allredj reviewed (commented):
Passed Diffblue compatibility checks (cbmc commit: ad054dcd31fd3a60b9afc055e5ad7c61b75b6269).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/87933977

thk123 commented on jbmc/CMakeLists.txt:
```diff
@@ -3,7 +3,7 @@ add_subdirectory(src)
 
 add_custom_target(java-models-library ALL
-    COMMAND mvn package
+    COMMAND mvn --quiet -Dmaven.test.skip=true package
```
:question: I'm with Chris - what is the reason for `Dmaven.test.skip=true` Doubt we have any tests in the core models and probably here isn't the place to run them if we do, but perhaps put this change in a separate commit so this can at least be justified.

tautschnig replied:
Many thanks, this should make it a lot clearer (and also easier to revert in future): done, please see whether the commit message is helpful.

tautschnig committed:
Make maven invocations quiet

The status messages make it unnecessarily hard to spot other compilation
errors.

tautschnig committed:
Maven: Disable attempts to run tests

There aren't any tests, and thus the build log is unnecessarily cluttered with

-------------------------------------------------------
 T E S T S
 -------------------------------------------------------

 Results :

 Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

chrisr-diffblue commented:
Split commits + commit messages look good to me - thanks for the extra clarification @thk123 and @tautschnig

allredj reviewed (commented):
Passed Diffblue compatibility checks (cbmc commit: 8f251b4cb24935b96c7b6aa8adda5c655e16a429).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/87939665
